test/parallel/test-util-inspect.js

```diff
@@ -263,19 +257,22 @@ assert.equal(util.inspect(a), '[ \'foo\', , \'baz\' ]');
 assert.equal(util.inspect(a, true), '[ \'foo\', , \'baz\', [length]: 3 ]');
 assert.equal(util.inspect(new Array(5)), '[ , , , ,  ]');
 
-// test for Array constructor in different context
-{
-  const Debug = require('vm').runInDebugContext('Debug');
-  const map = new Map();
-  map.set(1, 2);
-  const mirror = Debug.MakeMirror(map.entries(), true);
```

Comment:
May consider for future: We have a simulated Debug.MakeMirror in chakra_shim.js. Probably can support some of these.